navigator_engine/tests/util.py

```diff
@@ -132,3 +133,13 @@ def create_demo_data():
     ]
     model.db.session.add(graph_with_milestone)
     model.db.session.commit()
+
+
+def update_test_data_from_db():
+    graphs = model.Graph.query.all()
+    graph_storage = {}
+
+    for graph in graphs:
+        graph_storage[str(graph.id)] = graph.to_networkx()
+    with open(f'{app.config.get("TEST_DATA_GRAPH_FOLDER")}/graph_test_data.p', 'wb') as f:
+        pickle.dump(graph_storage, f)
```

Comment:
I can't see where this function is called elsewhere in the PR?

Reply:
It's not called anywhere. I've included it to be run manually when the pickle test data storage needs to be updated. We can also include it as a separate script.

Reply:
Ok I see, can you add a comment into the function explaining intended use of the function?